tests/testapp/migrations/0001_initial.py

```diff
@@ -92,193 +92,123 @@ class Migration(migrations.Migration):
                     "body",
                     wagtail.fields.StreamField(
                         [
-                            (
-                                "heading",
-                                wagtail.blocks.CharBlock(form_classname="full title"),
+                            ("heading", 0),
```

Comment:
this looks like the Wagtail 6.2 changes - https://docs.wagtail.org/en/latest/releases/6.2.html#compact-streamfield-representation-for-migrations

What version did you run the migrations with? It doesn't matter as such because they are more logical representations, and we don't do anything with SF migration

Reply:
Yep, I've generated them with Wagtail 6.2. But the test matrix is passing, so it looks like older Wagtail versions can still read them.